extensions/guacamole-auth-quickconnect/src/licenses/bundled/aopalliance-1.0/LICENSE

```diff
@@ -0,0 +1,4 @@
+From http://aopalliance.sourceforge.net/:
```

Comment:
Is this still bundled? Looking at the `pom.xml`, I no longer see any dependencies that are not marked as provided.